### ISSUE TITLE
Peer to Peer code Review on Validate Contact Form

### ISSUE THREAD

**Issue by Abenezer-Tilahun** [AckonSamuel/Hello-Portfolio#18]
You have done a good job in this milestone 👍🏻! I love how simple you wrote the code to accomplish Validate form. Keep coding, keep soaring high 🚀!
Your project is complete! There is nothing else to say other than... it's time to merge it 
Congratulations! 🎉

**Comment by beyk** [comment]
Yeah, It was nice coding, without extra lines. Congratulations man/

**Comment by stanleySimeon** [comment]
# Hi @beyk,
You've done great work in your PR!

Your project is complete! There is nothing else to say other than... it's time to merge it Congratulations! 🎉 🎉 🎉

Feel free to ask any questions.

Cheers and happy coding!
